morse_decoder.rb

```diff
@@ -0,0 +1,39 @@
+# decode_char
+def decode_char(char)
+  morse_dict = {
+    '.-' => 'a', '-...' => 'b', '-.-.' => 'c', '-..' => 'd',
+    '.' => 'e', '..-.' => 'f',
+    '--.' => 'g', '....' => 'h',
+    '..' => 'i', '.---' => 'j',
+    '-.-' => 'k', '.-..' => 'l',
+    '--' => 'm', '-.' => 'n',
+    '---' => 'o', '.--.' => 'p',
+    '--.-' => 'q', '.-.' => 'r',
+    '...' => 's', '-' => 't',
+    '..-' => 'u', '...-' => 'v',
+    '.--' => 'w', '-..-' => 'x',
+    '-.--' => 'y', '--..' => 'z',
+    ' ' => ' ', '.----' => '1',
+    '..---' => '2', '...--' => '3',
+    '....-' => '4', '.....' => '5',
+    '-....' => '6', '--...' => '7',
+    '---..' => '8', '----.' => '9', '-----' => '0'
+  }
+  morse_dict[char]
+end
+
+def decode_word(str)
+  string = str.split
+  arr = []
+  string.each { |i| arr.push(decode_char(i).to_s) }
+  arr.join(' ')
+end
+
+def decode_phrase(str)
+  phrase = str.split('   ')
+  arr = []
+  phrase.each { |i| arr.push(decode_word(i).to_s) }
+  arr.join('   ')
+end
```

Comment:
- [ ] Kindly when I try to convert it gives me a space between each character, kindly fix that to only put a space between only words.

![image](https://user-images.githubusercontent.com/35535130/180272425-f6e15a1f-522b-42cb-b6ce-42edf8692942.png)